tests/caesarlib.rs

```diff
@@ -51,4 +51,12 @@ mod tests {
         let (_, enciphered) = rdm_encipher(test_word);
         assert_ne!(test_word, &enciphered);
     }
+
+    #[test]
+    fn custom_base_sequence_correct_encipherd_and_deciphered() {
+        let base = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz".chars().rev().collect::<String>();
```

Comment:
Since custom_encipher and custom_decipher takes a str, you don't need to do all of that. Just stick with the string it should be enough.